### PR TITLE
Slack selector component

### PR DESCRIPTION
**Pull request by bago2k4**
card: https://trello.com/c/vSNRM6c1

Test:
- run `lein test-ui-components!`
- open a report page
- check at the bottom the rendered selector
- try to select a different value
- try the search field

### CODE REVIEW

File: src/open_company_web/components/user_selector.cljs [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        img (.data el "icon")
+        my-temp (.$ js/window (str "<span><img class=\"user-icon\" src=\"" img "\" />" (.-text state) "</span>"))]
+    my-temp))
+
+(defcomponent user-selector [data owner]
+  (will-mount [_]
+    (cdr/add-script! "//ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js")
+    (cdr/add-script! "/lib/select2/js/select2.js")
+    (cdr/add-style! "/lib/select2/css/select2.css"))
+  (did-mount [_]
```
Comment (belucid) [inline src/open_company_web/components/user_selector.cljs]: Could stand to have a comment here about what's going on and why

**Comment by bago2k4** [comment]
@belucid done

**Comment by belucid** [comment]
@bago2k4 In addition to the callback rather than timeout, filter out bots completely rather than disable.

Ping me when it's rdy for another review.

**Comment by bago2k4** [comment]
@belucid ready to go

**Pull request merged**